package/gen_versions.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"html/template"
+	"os"
+
+	"github.com/rancher/types/apis/management.cattle.io/v3"
```

Comment:
where is the vendor dependency being defined? is it coming as a part of a separate commit?